tool/tsh/tsh.go

```diff
@@ -3786,10 +3786,16 @@ func onShow(cf *CLIConf) error {
 // printStatus prints the status of the profile.
 func printStatus(debug bool, p *profileInfo, env map[string]string, isActive bool) {
 	var prefix string
-	duration := time.Until(p.ValidUntil)
 	humanDuration := "EXPIRED"
+	duration := time.Until(p.ValidUntil)
 	if duration.Nanoseconds() > 0 {
-		humanDuration = fmt.Sprintf("valid for %v", duration.Round(time.Minute))
+		// Display <1m if less then 1 minute left while valid instead of 0s
+		oneMinuteAhead := time.Now().UTC().Add(1 * time.Minute)
```

Comment:
This looks super complicated.

Why not just `if duration < 1 * time.Minute`?

Reply:
done, thanks!